internal/commands/loglevel/show_test.go

```diff
@@ -52,7 +59,7 @@ var _ = Describe("Show", func() {
 		c := fake.NewClientBuilder().WithScheme(scheme).WithObjects(initObjects...).Build()
 
 		utils.InitalizeManagementClusterAcces(scheme, nil, nil, c)
-		err = loglevel.ShowLogSettings(context.TODO())
+		err = loglevel.ShowLogSettings(context.TODO(), testNamespace, testClusterName)
```

Comment:
loglevel.ShowLogSettings(context.TODO(), "", "") so we can test when looking at the management cluster it works and

loglevel.ShowLogSettings(context.TODO(), testNamespace, testClusterName) to verify in a managed cluster

Reply:
Should I also add the _clusterType_ to the tests or keep it at those two?

Reply:
We need ShowLogSettings to accept clusterType as well. But for testing we can pass empty values.